nuxeo-tools-hooks/nxtools/hooks/tests/webhooks/github_handlers/test_pullrequest_review.py

```diff
@@ -172,7 +172,7 @@ def test_store_pull_request_trigger_review_private(self):
         self.setup_services_config()
         with GithubHookHandlerTest.payload_file('github_pullrequest_open') as payload, \
             GithubHookHandlerTest.payload_file('github_pullrequest') as prp, \
-            GithubHookHandlerTest.payload_file('github_pullrequest_commits') as prc:
+            GithubHookHandlerTest.payload_file('github_pullrequest_commits_2') as prc:
```

Comment:
will avoid adding a second file for just the commit message change (wanted to unit-test the single jira issue too)